api/apiclient.go

```diff
@@ -1299,6 +1305,10 @@ func (s *state) IPAddr() string {
 	return s.ipAddr
 }
 
+func (s *state) IsProxied() bool {
```

Comment:
either comment or lowercase
does it need to be exported?

Reply:
Done & Done